script/update-json-schema.sh

```diff
@@ -1,5 +1,26 @@
 #!/bin/bash
 
-NEW_HASH=$(git ls-remote git://github.com/department-of-veterans-affairs/vets-json-schema/ refs/heads/master | awk '{print $1}')
-sed -i '' -e 's/\(vets-json-schema\.git#\)\(.*\)\(",$\)/\1'$NEW_HASH'\3/' package.json
+# Find the versions
+NEW_VERSION=$(git ls-remote --tags git://github.com/department-of-veterans-affairs/vets-json-schema/ | awk -F / '{ print $3 }' | sort -V -r | head -n 1)
+OLD_VERSION=$(grep -oP '(?<=department-of-veterans-affairs/vets-json-schema#)(.*)(?=",$)' package.json)
+
+# Set up color output
+GREEN='\033[0;32m'
+BROWN='\033[0;33m'
+RESET='\033[0m'
+NEW_VER_OUT="${GREEN}$NEW_VERSION${RESET}"
+OLD_VER_OUT="${BROWN}$OLD_VERSION${RESET}"
+
+# Escape hatch
+if [ "$NEW_VERSION" == "$OLD_VERSION" ]; then
+    echo -e "Newest version found on GitHub matches the version currently in package.json ($NEW_VER_OUT)"
+    exit 0
+fi
+
+echo -e "Updating vets-json-schema from $OLD_VER_OUT to $NEW_VER_OUT"
+
+REPLACE_COMMAND='s/\(department-of-veterans-affairs\/vets-json-schema#\)\(.*\)\(",$\)/\1'$NEW_VERSION'\3/'
+
+# `sed -i ''` works for BSD sed, but not GNU; fall back to GNU
+sed -i '' -e "$REPLACE_COMMAND" package.json 2>/dev/null || sed -i -e "$REPLACE_COMMAND" package.json
```

Comment:
Only problem I can see with this is if `sed` fails for some reason that doesn't stem from the BSD / GNU mismatch. That might just be if `package.json` isn't there (in which case, the second command will likewise fail).

It _should_ be okay, though. :crossed_fingers: